packages/models-library/src/models_library/services_types.py

```diff
@@ -59,6 +72,17 @@ def create(cls) -> "RunID":
         run_id_format = f"{utc_int_timestamp}_{uuid4()}"
         return cls(run_id_format)
 
+    @classmethod
+    def get_resource_tracking_run_id(
```

Comment:
Proposal: maybe lets have:
 - `get_resource_tracking_run_id_for_computational`
 - `get_resource_tracking_run_id_for_dynamic`
    - here you can just call the function you are using to generate the dynamic

Reply:
The concept is the same for both computational and dynamic. I think it's good for using inside the resource_tracking. Also the dy_sidecar uses it to distinguish between runs which is OK